src/docs/Composing_Applications.mdx

```diff
@@ -40,23 +40,19 @@ Create `package.json` in this directory:
 {
   "private": true,
   "dependencies": {
-    "typescript": "latest",
-    "@theia/typescript": "next",
+    "@theia/callhierarchy": "next",
+    "@theia/file-search": "next",
+    "@theia/git": "next",
+    "@theia/json": "next",
+    "@theia/markers": "next",
+    "@theia/messages": "next",
+    "@theia/mini-browser": "next",
     "@theia/navigator": "next",
-    "@theia/terminal": "next",
     "@theia/outline-view": "next",
     "@theia/preferences": "next",
-    "@theia/messages": "next",
-    "@theia/git": "next",
-    "@theia/file-search": "next",
-    "@theia/markers": "next",
     "@theia/preview": "next",
-    "@theia/callhierarchy": "next",
-    "@theia/merge-conflicts": "next",
     "@theia/search-in-workspace": "next",
-    "@theia/json": "next",
-    "@theia/textmate-grammars": "next",
-    "@theia/mini-browser": "next"
+    "@theia/terminal": "next"
```

Comment:
It would be good to include plugin-ext-vscode and built-in plugins.

Reply:
I updated the code to include `@theia/plugin-ext-vscode` and also provide an example with using the builtins.